packages/fish.rb

```diff
@@ -46,6 +46,17 @@ def self.install
   end
 
   def self.postinstall
+    puts 'Installing Oh My Fish...'.lightgrey
+    system 'curl https://raw.githubusercontent.com/oh-my-fish/oh-my-fish/master/bin/install | fish'
```

Comment:
We should be using the version tag of this package instead of the master branch.  In fact, I think we should be using fish instead of oh-my-fish, no?

Reply:
Fish dosen't provide fenv, which is the cleanest way to source `~/.profile`.

Reply:
Would it make sense to create a separate oh_my_fish package and add a dependency to this package?

Reply:
That would create a cyclic dependency, no?